src/GameLoop.cpp

```diff
@@ -29,6 +29,8 @@ static RageTimer g_GameplayTimer;
 
 static Preference<bool> g_bNeverBoostAppPriority( "NeverBoostAppPriority", false );
 
+int CheckInputDevicesCounter = 0;
```

Comment:
You can make this a static variable within the RunGameLoop function. It's a bit nicer since keeps the variable closer to its usage.

Reply:
Good idea :)